src/Jira/Api.php

```diff
@@ -68,27 +68,34 @@ class Api
 	protected $options = self::AUTOMAP_FIELDS;
 
 	/**
-	 * Fields.
+	 * Client-side cache of Fields. List of fields when loaded, null when nothing is fetched yet.
```

Comment:
The `Fields` should be in lowercase. It was capitalized because it was 1st word in sentence.